src/Umbraco.Core/Configuration/Models/ConnectionStrings.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace Umbraco.Core.Configuration.Models
+{
+    public class ConnectionStrings
+    {
+        // Backing field for UmbracoConnectionString to load from configuration value with key umbracoDbDSN.
+        // Attributes cannot be applied to map from keys that don't match, and have chosen to retain the key name
+        // used in configuration for older Umbraco versions.
+        // See: https://stackoverflow.com/a/54607296/489433
+        private string umbracoDbDSN
+        {
+            get => UmbracoConnectionString?.ConnectionString;
```

Comment:
[minor] Can't see where this is used yet

Reply:
The aspnetcore config used reflection to populate the properties from the config file. More info on the link in the comment